### ISSUE TITLE
[NEOS Discovery] Cross reference NEOS Discovery changelog against UAL Discovery changelog

### ISSUE THREAD

**Issue by seanluyk** [ualbertalib/NEOSDiscovery#270]
To ensure we haven't forgotten to apply improvements/bug fixes made in UAL discovery to NEOS discovery, cross-check changelogs and apply code changes as possible.

**Comment by theLinkResolver** [comment]
@seanluyk yes please. Just noticed that finding aid links are not displaying in neos discovery.
